### PR TITLE
add mobile styles to app launcher

### PR DESCRIPTION
**Pull request by rpanichakit9541**
This is for this home app issue https://devtopia.esri.com/WebGIS/arcgis-portal-app/issues/31145

### CODE REVIEW

File: src/header/css/header-apps.pcss [hunk 1]
```diff
@@ -169,6 +169,10 @@ html[dir="rtl"] .dropdown-menu.dropdown-right {
 
 .esri-header-apps .dropdown-width-3 {
 	min-width: 480px;
+
+	@media (width <= 480px) {
+		min-width: 350px;
```
Comment (paulcpederson) [inline src/header/css/header-apps.pcss:174]: So on really small phones (320px) this will side scroll, is that desired?

Comment (rpanichakit9541) [reply src/header/css/header-apps.pcss:174]: No I don't think that's the desired behavior.  Do we add another media query for (width <= 350px) or is there a better way to make this work better overall?

Comment (paulcpederson) [reply src/header/css/header-apps.pcss:174]: You could play with sizing to `100vw` or something (could also use the `--esir-vw` unit maybe?)

**Comment by rpanichakit9541** [comment]
Thanks for the suggestions @paulcpederson . I've updated the styles to make them work more universally.

**Review by paulcpederson — APPROVED** [review]

**Pull request merged**